src/graph/serialize/heterograph_serialize.cc

```diff
@@ -1,7 +1,7 @@
 /**
  *  Copyright (c) 2019 by Contributors
  * @file graph/serialize/heterograph_serialize.cc
- * @brief DGLHeteroGraph serialization implementation
+ * @brief DGLGraph serialization implementation
  *
  * The storage structure is
  * {
```

Comment:
In Line 11, there is kDGLHeteroGraph, should that be replaced or already outdated?

Reply:
I suggest not touching `.cc` files for the time being.

Reply:
Revert C code.